bin/test_exercises.sh

```diff
@@ -66,7 +74,7 @@ do
     if [ "$compile_exit_code" -eq 0 ]
     then
       # perform unit tests
-      test_results=$(mix test --no-compile --color --no-elixir-version-check --include pending 2> /dev/null)
```

Comment:
Running the test with `--no-compile` makes tests fail for bread-and-potions which requires that protocols are consolidated.